docs/advanced-topics/legacy/legacy-getting-started-with-cwl.rst

```diff
@@ -0,0 +1,383 @@
+.. note:: This tutorial is a continuation of :doc:`Getting Started With Docker </getting-started/getting-started-with-docker>`. Please complete that tutorial prior to doing this one.
+
+Getting Started with CWL (Legacy)
+=================================
+
+Tutorial Goals
+--------------
+
+-  Learn about the `Common Workflow Language
+   (CWL) <https://www.commonwl.org/user_guide/>`__
+-  Create a basic CWL Tool which uses a Docker image
+-  Run the Tool locally
+-  Describe a sample parameterization of the Tool
+-  Push the Tool onto GitHub
+
+Tutorial Background
+-------------------
+This is a legacy version of our "Getting Started with CWL" tutorial. As is mentioned in :doc:`Overhauling Dockstore Tools </advanced-topics/dockstore-tools-overhaul>`, our definition of a tool has changed over time. This tutorial was designed for the older definition, where we required people to associate a tool with a Dockerfile. Strictly speaking, you can still follow this tutorial to end up with a valid CWL tool and register it as a GitHub App tool, but the work you are doing in this tutorial goes above and beyond what we now require for CWL tools. You can find a simplier version of this tutorial at :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>`.
```

Comment:
typo: simpler instead of simplier

Reply:
Suggest rephrasing for succinctness: You can still follow this tutorial to create a valid CWL tool and register it as a GitHub App tool, but we recommend following a simpler version of this tutorial at :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>`.